en/advanced_features/traffic_avoidance_utm.md

```diff
@@ -0,0 +1,34 @@
+# Air Traffic Avoidance: MAVLink
+
+PX4 can use MAVLink messages to support simple air traffic avoidance in [missions](../flight_modes/mission.md).
+If a potential collision is detected, PX4 can *warn*, immediately [land](../flight_modes/land.md), or [return](../flight_modes/return.md) (depending on the value of [NAV_TRAFF_AVOID](#NAV_TRAFF_AVOID)).
+
+This implementation is based on the [ADS-B traffic avoidance](../advanced_features/traffic_avoidance_adsb.md). The `UTM_GLOBAL_POSITION` MAVLink
+message is treated exactly the same as an ADS-B message.
+
+
+## Configure Traffic Avoidance
+
+Configure the action when there is a potential collision using the parameter below:
+
+Parameter | Description
+--- | ---
+<span id="NAV_TRAFF_AVOID"></span>[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Enable traffic avoidance mode specify avoidance response. 0: Disable, 1: Warn only, 2: Return mode, 3: Land mode.
+
+
+## Implementation
```

Comment:
Maybe add explicit anchor

```
## Implementation {#implementation}
```

FYI, the anchor format above works in gitbook but not github. We don't HAVE to do this - an anchor is created for every heading, and it will have the same format. What this does is allow us to later change the heading without changing anchors.